docs/tutorial.md

```diff
@@ -425,7 +425,7 @@ To remove specific listener you can use `bot.removeListener()` method.
 - `bot.once(eventName, listener)`
   Execute the `listener` function, only once, the first time the event named `eventName` triggered.
 - `bot.removeListener(eventName, listener)`
-  Removes the specified `listener` for the event named `eventName`. This will not work with annonymous functions, as they don't have a name.
+  Removes the specified `listener` for the event named `eventName`. This will only work with named functions `function myNamedFunc() {}`
```

Comment:
That is not true, discussion [here](https://github.com/PrismarineJS/mineflayer/pull/1148#discussion_r454716261)

Reply:
suggestion for @Naomi-alt , put this : 
> Removes the specified `listener` for the event named `eventName`. In order to use this you either need to define your function with `function myNamedFunc() {}` or put your function in a variable with `const myNamedFunc = () => {}`. You can then use myNamedFunc in the `listener` argument

suggestion for @wvffle : I know it's clear for you but it doesn't seem clear for @Naomi-alt , sometimes suggesting a fix is a good way to help understand and unlock the situation. Reviews are meant to help improve the quality of pull requests, not to block them and we should do our best to reach that objective.

Reply:
Changed it.
I guess my confusion came from the fact that anonymous functions assigned to a variable are still anonymous, while I thought they weren't.
Thank for clearing it up.